packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -708,7 +724,8 @@ $ms-Persona-presenceSizeXl: 28px;
       background-color: $ms-color-presence-busy-average;
       transform: rotate(-45deg);
       position: absolute;
-      top: 5px;
+      // top: 5px;
+      top: 40%;
```

Comment:
Where does 40% come from? Would be good to include a comment here explaining. Can also remove the commented-out line above.